tests/installation/bootloader.pm

```diff
@@ -236,15 +242,15 @@ sub run() {
         $args .= " autoupgrade=1";
     }
 
-    type_string $args, 13;
+    type_string $args, $slower_typing_speed;
```

Comment:
these are the values that really change, all in before are just extracting variables.